src/test/scala/spray/json/JsonParserSpec.scala

```diff
@@ -74,7 +74,8 @@ class JsonParserSpec extends Specification {
     }
     "be reentrant" in {
       val largeJsonSource = scala.io.Source.fromInputStream(getClass.getResourceAsStream("/test.json")).mkString
-      List.fill(20)(largeJsonSource).par.map(JsonParser(_)).toList.map {
+      import scala.collection.parallel.immutable.ParSeq
+      ParSeq.fill(20)(largeJsonSource).map(JsonParser(_)).toList.map {
```

Comment:
(in 2.13 you need to import an implicit before you can do `.par`, but in 2.12 the import won't compile, so the easiest way to cross-build is just to avoid `.par`)